toolsrc/src/vcpkg/commands.fetch.cpp

```diff
@@ -18,6 +20,36 @@ namespace vcpkg::Commands::Fetch
     {
         (void)args.parse_arguments(COMMAND_STRUCTURE);
         const std::string tool = args.command_arguments[0];
+
+        if (tool == "update")
+        {
+            const fs::path& git_exe = paths.get_tool_exe(Tools::GIT);
+            const std::string repo = "https://github.com/microsoft/vcpkg";
+            fs::path fetch_dir = paths.root / fs::u8path("vcpkg-fetch");
+
+            Files::Filesystem& fs = paths.get_filesystem();
+
+            fs.remove_all(fetch_dir, ignore_errors);
+            fs.create_directory(fetch_dir, VCPKG_LINE_INFO);
+
+            const std::string full_cmd = Strings::format(
+                R"(%s clone %s %s)", fs::u8string(git_exe), fs::u8string(repo), fs::u8string(fetch_dir));
```

Comment:
```suggestion
                R"("%s" clone "%s" "%s")", fs::u8string(git_exe), fs::u8string(repo), fs::u8string(fetch_dir));
```

This needs to be made resistant to spaces in the path

Reply:
I believe we have an "escape for command line" function.